test/server/http/admin_test.cc

```diff
@@ -335,6 +337,218 @@ TEST_P(AdminStatsTest, UsedOnlyStatsAsJson) {
   store_->shutdownThreading();
 }
 
+TEST_P(AdminStatsTest, StatsAsJsonFilterString) {
```

Comment:
Can we also add one integration test for this here https://github.com/envoyproxy/envoy/blob/master/test/integration/integration_admin_test.cc#L155

Reply:
Thanks, see 4340178.